wiki/versions/1.19/getting_started/setup/markdown/setup.md

```diff
@@ -0,0 +1,101 @@
+it.
+
+- [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack): A package of Java-related
+extensions for VSCode.
+
+Then open the folder as a project and run the `vscode` Gradle task.
+
+### Customizing your Project
+
+Even though you have a ready workspace, it still has all the default values from the template, which is no good.
+You should replace them to match the details of your mod.
+
+- `build.gradle`
+
+    The Gradle buildscript, it specifies dependencies, packaging, compiling, and publishing behavior.
+
+- `gradle.properties`
+
+    Properties that can be used in the buildscript.
+    The default values included in it, that you should modify first are: `version`, the current version of the mod;
+`maven_group`, the group ID for your mod, preferable it should follow Java's package convention, that is, the reverse
+of a domain you own (e.g. `mydomain.com` -> `com.mydomain`), alphanumeric lowercase with underscores replacing dashes;
+`archives_base_name`, the name of the built JAR file.
+
+- `libs.versions.toml`
+
+    Specifies the dependency notation of certain dependencies of your mod, by default Minecraft, Quilt Mappings, Quilted
+Fabric API, and Quilt Loader.
+
+    They might not be always up-to-date, if that is the case, check the [Quilt Import Utility Page](https://lambdaurora.dev/tools/import_quilt.html).
+
+    In case of adding new dependencies, don't forget to also add them and their repositories into your buildscript with a
+configuration befit for them.
+
+- `LICENSE`
+
+    A default `LICENSE` file is not included by itself in the template mod, only a template, choose a license that matches
+your own priorities and thoughts, and then create a file that has the legal text for it.
+
+    You also should set the `license` field in the `quilt.mod.json` to the [SPDX Identifier](https://spdx.org/licenses) of the chosen license.
+
+- `quilt.mod.json`
+
+    The Quilt mod metadata file.
+    Contains a variety of information relating to your mod.
+
+    A complete specification of all the fields and format can be accessed in [the RFC](https://github.com/QuiltMC/rfcs/blob/main/specification/0002-quilt.mod.json.md).
+
+- `File Structure`
+
+    The packages of source should match the group inserted into the `gradle.properties`.
+
+### Generating the Minecraft Source
+
+While Minecraft is not open-source nor readable in normal circumstances, people have found ways to decompile, unobfuscate
+and remap the game.
+
+Although not strictly necessary, generating sources allow for facility in navigating through the code and for a more
+pleasant code viewing experience.
+
+The default of Quilt Loom - The Gradle plugin that the template mod uses - has three decompilers for choosing, Quiltflower
+(Notably, a Quilt project), CFR, and Fernflower.
+
+Generating the source is a simple as running the `genSources` (Append `with[Decompiler Name]` in case you want a decompiler
```

Comment:
Actually, something tells me that CFR is being chosen by default for some reason, soooooooo
although something also tells me that QLoom should fix this

Reply:
If it is, then the patch is not being properly applied:
https://github.com/QuiltMC/quilt-loom/blob/bad86783989d76cb3f5f23fe6d7b2bb44c303bac/patches/0004-Add-support-for-other-Quilt-projects-and-the-Quilt-m.patch#L85-L86

Reply:
Very, very interesting;
Actually, I should become one with the elephant; I'll investigate that issue later